data/pigui/modules/flight-ui/reticule.lua

```diff
@@ -321,8 +322,10 @@ local function displayDetailData(target, radius, colorLight, colorDark, tooltip,
 	local uiPos = ui.pointOnClock(center, radius, 2.46)
 	-- label of target
 	local nameSize = ui.addStyledText(uiPos, ui.anchor.left, ui.anchor.baseline, target.label, colorDark, pionillium.medium, tooltip, colors.lightBlackBackground)
-	if ui.isMouseHoveringRect(uiPos - Vector2(0, pionillium.medium.size), uiPos + nameSize - Vector2(0, pionillium.medium.size)) and ui.isMouseClicked(1) and ui.noModifierHeld() then
-		ui.openDefaultRadialMenu("game", target)
+	if ui.isMouseHoveringRect(uiPos - Vector2(0, pionillium.medium.size), uiPos + nameSize - Vector2(0, pionillium.medium.size)) and ui.isMouseClicked(1) and ui.noModifierHeld()
```

Comment:
I would strongly prefer that the `ui.isMouseHoveringRect()` part of the condition be split into another temporary variable named `isHovered`, as with the introduction of the `or` case it's quite lengthy and bug-prone.

Reply:
Fixed in 894b7ad45be1c7494907132406299a3ed3a431fc